src/intel/RoomIntel.ts

```diff
@@ -351,9 +353,25 @@ export class RoomIntel {
 		return 0;
 	}
 
+	static requestZoneData() {
+		const checkOnTick = 123;
+		if (Game.time % 1000 == checkOnTick - 2) {
+			Segmenter.requestForeignSegment('LeagueOfAutomatedNations', 96);
+		} else if (Game.time % 1000 == checkOnTick - 1 ) {
+			const loanData = Segmenter.getForeignSegment();
+			if (loanData) {
+				Memory.zoneRooms = loanData;
+			} else {
+				log.error('Empty LOAN data');
+			}
+		}
+	}
+
 
 	static run(): void {
 		let alreadyComputedScore = false;
+		this.requestZoneData();
```

Comment:
if `requestZoneData` is static (per line 356 above), wouldn't it be `RoomIntel.requestZoneData()`, not `this.requestZoneData()`?

Reply:
Naw, the rest of the static access is
```
				this.updateInvasionData(room);
				this.updateHarvestData(room);
				this.updateCasualtyData(room);
```